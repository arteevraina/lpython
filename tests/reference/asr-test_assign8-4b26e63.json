{
    "basename": "asr-test_assign8-4b26e63",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_assign8.py",
    "infile_hash": "e7f5e3dd10847a04fc4634a70fb7908f4360a0b71c1f6718f7beeee9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_assign8-4b26e63.stderr",
    "stderr_hash": "900fcd6359806b1e7ab41b6c5c4eb252bdbab6e96404912b710a8682",
    "returncode": 2
}