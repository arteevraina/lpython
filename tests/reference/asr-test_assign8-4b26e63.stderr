warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/errors/test_assign8.py:5:13
  |
5 |     c = c32(complex(3, 4))
  |             ^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
